pkgs/biomejs/biome/pkg.yaml

```diff
@@ -4,3 +4,5 @@ packages:
     version: "@biomejs/biome@2.0.0-beta.5"
   - name: biomejs/biome
     version: "@biomejs/biome@2.0.0-beta"
+  - name: biomejs/biome
+    version: "@biomejs/biome@2.0.4"
```

Comment:
This is unnecessary.
The first element `- name: biomejs/biome@cli/v1.9.4` would be updated by aqua-registry-updater.